client/src/app/page.js

```diff
@@ -1,10 +1,33 @@
 import Image from 'next/image'
 import styles from './page.module.css'
+import React from 'react';
+import logo from 'public/images/logo.png';
+import Button from '@/components/button-component/button';
+import buttonStyle from '@/components/button-component/button.module.css';
+import Divider from '@/components/divider-component/divider';
 
 export default function Home() {
   return (
     <main className={styles.main}>
-      <h1>Welcome to Cultivate</h1>
+        <div className={styles.landing}>
+            <h1 className={styles.welcome}>welcome to <span style={{ color: '#2B7E59' }}>cultivate!</span></h1>
+            <p className={styles.message}><em>cultivate</em> is designed to help make your happiness a habit. 
+            write down a happy memory and grow your garden!</p>
+            <div className={styles.buttonContainer}>
+                <Button className={buttonStyle.signUp}>Sign Up</Button>
```

Comment:
Suggestion: you don't need to change this, but you can add a variant prop to your Button to change the color like in this [article](https://medium.com/cstech/building-reusable-ui-components-with-react-best-practices-and-patterns-24b6fe921347)